### PR TITLE
Glossary changes DOC-1542

### PR DESCRIPTION
**Pull request by kaitlynmichael**
DOC-1542
Verified or added the following terms:

high availability
multi-primary
replicaHA
clustering
replication
consistency
syncer
distributed sync
proxy policy
data persistence
participating clusters
replication backlog
FQDN
mDNS
tombstone

### CODE REVIEW

**Review by cmilesb — APPROVED** [review]
Looks good to me!

File: content/glossary/_index.md [hunk 1]
```diff
@@ -132,25 +148,32 @@ More info: [`fsync` man page]("https://man7.org/linux/man-pages/man2/fsync.2.htm
...
 
-More info: [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/)
+More info: [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/), [Ingress routing for Redis Enterprise Software on Kubernetes]({{<relref "/kubernetes/re-databases/set-up-ingress-controller.md">}})
 {{%/definition%}}
 
 ## K, L {#letter-k}
 
 {{%definition "kubectl"%}}
-A command line tool for communicating with a Kubernetes API server. 
+A command line tool for communicating with a Kubernetes API server.
```
Comment (rrelledge) [inline content/glossary/_index.md:176]:
```suggestion
A command-line tool for communicating with a Kubernetes API server.
```

File: content/glossary/_index.md [hunk 2]
```diff
@@ -163,23 +186,32 @@ More info: [LDAP wikipedia](https://en.wikipedia.org/wiki/Lightweight_Directory_
...
+{{%definition "master node"%}}
+Node that operates as the leader of a cluster. Also known as the primary node.
 {{%/definition%}}
 
 {{%definition "migration"%}}
 Deciding when and where shards will be moved if more network throughput, memory, or CPU resources are needed
 {{%/definition%}}
 
+{{%definition "multicast DNS (mDNS)"%}}
+Protocol that resolves hostnames to the IP adressess that do not include a local name server.
```
Comment (rrelledge) [inline content/glossary/_index.md:198]:
```suggestion
Protocol that resolves hostnames to the IP addresses that do not include a local name server.
```

File: content/glossary/_index.md [hunk 3]
```diff
@@ -267,21 +309,30 @@ Enables your Redis databases to span both RAM and dedicated flash memory (SSD).
 More info: [Redis on Flash]({{<relref "/rs/concepts/memory-performance/redis-flash.md">}}), [Getting Started with Redis on Flash (RoF)]({{<relref "/rs/administering/cluster-operations/getting-started-redis-flash.md">}})
 {{%/definition%}}
 
+{{%definition "replica high availability (replicaHA)"%}}
+High availability feature of Redis Enterprise Software. After a node failure, the cluster automatically migrates remaining replica shards available nodes. Previously known as "Slave HA" or `slave_ha`.
```
Comment (rrelledge) [inline content/glossary/_index.md:313]:
```suggestion
High availability feature of Redis Enterprise Software. After a node failure, the cluster automatically migrates remaining replica shards to available nodes. Previously known as "Slave HA" or `slave_ha`.
```

**Review by rrelledge — APPROVED** [review]
Looks good to me. I just left a couple of suggestions for typo fixes.

**Pull request merged**